content/maxwell3_fdem/emi_tutorial/induced_currents_body.rst

```diff
@@ -0,0 +1,94 @@
+.. _induced_currents_body:
+
+Induced currents in the body
+============================
+
+Consider a simple equivalent circuit as shown in :numref:`Concepts_3loops_only`.
+
+.. figure:: ./images/Concepts_3loops_only.png
+   :align: center
+   :scale: 50%
+   :name: Concepts_3loops_only
+
+   Conceptual diagram for 3-loops system.
+
+Through the Tx (Loop1) we flow time-varying currents, then it generates time varying magnetic flux, which will exite the body (Loop2). Let :math:`\Phi^{p}` be the primary magnetic flux from the Tx. The body treats like and R-L circuilt.
```

Comment:
grammar is a bit off. I would suggest something along the lines of
```
A time-varying current is put through the Tx (loop 1), which then generates a time varying magnetic flux (primary
magnetic flux), which we denote :math:`\Phi^{p}`. If coupled, this will induce currents in the body (Loop2) according to 
:ref:`Lenz's Law<lenz_law>`. We can examine the nature of the induced current by treating the loop as a Resistor-
Inductor (RL) circuit.
```

I also think adding an image to show coupling here would be beneficial. Something along the lines of this?  
<img width="367" alt="emlabintroslides pptx 2016-04-12 20-04-53" src="https://cloud.githubusercontent.com/assets/6361812/14481664/0829ad9e-00ea-11e6-959b-ea4433800632.png">